src/b3d/utils.py

```diff
@@ -29,7 +29,7 @@
 
 
 def get_root_path() -> Path:
-    return Path(Path(b3d.__file__).parents[1])
+    return Path(Path(b3d.__file__).parents[2])
```

Comment:
this is the ONE change that gets the test passing, other than the setup.py changes.